tests/by-util/test_cp.rs

```diff
@@ -3956,7 +3956,7 @@ fn test_cp_default_virtual_file() {
     let ts = TestScenario::new(util_name!());
     let at = &ts.fixtures;
     ts.ucmd()
-        .arg("/sys/kernel/address_bits")
+        .arg("/sys/kernel/profiling")
```

Comment:
Please add a comment in the Rust code explaining why you chose this file, hust like you already explained on Discord. My first attempt would be:

```suggestion
    ts.ucmd()
        // This file has existed at least since 2008, so we assume that it is present on "all" Linux kernels.
        // https://www.kernel.org/doc/Documentation/ABI/testing/sysfs-profiling
        .arg("/sys/kernel/profiling")
```